### PR TITLE
Add declining price chart to collectible details with recharts

### PR DESCRIPTION
**Pull request by mariano-aguero**
No issue related

#### Added:

- Recharts to the package json ([MIT License](https://github.com/recharts/recharts/blob/master/LICENSE) )
- Add a chart to a collectible if is dutch auction
Should look like this:

Screenshot_20190523_144610

### CODE REVIEW

**Comment by mariano-aguero** [comment]
Hi @chriskalani 
This graph implementation uses [Recharts](http://recharts.org/en-US/), with a MIT LICENSE. Do you agree that we use this library? Any thoughts ? 

Regards

**Review by unjapones — APPROVED** [review]

File: src/components/erc721/common/declining_price_graph.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                        dataKey="markerLimit"
+                        stroke="#00AE99"
+                        fill="#00AE99"
+                        dot={{ stroke: '#00AE99', strokeWidth: 4 }}
+                    />
+                    <Area
+                        type="monotone"
+                        dataKey="currentPrice"
+                        stroke="#00AE99"
+                        fill="#00AE99"
```
Comment (Agupane) [inline src/components/erc721/common/declining_price_graph.tsx]: R: Could you put is values in as constants?

**Comment by chriskalani** [comment]
Nice! Let's go with this

**Comment by Agupane** [comment]
The chart looks awesome for me, I just added a comment, but it's non-blocking.

File: src/components/erc721/common/declining_price_graph.tsx (hunk 1 above)
Comment (mariano-aguero) [reply src/components/erc721/common/declining_price_graph.tsx]: Added the color values to the theme properties

**Review by Agupane — APPROVED** [review]

**Pull request merged**